app/api/websocket.py

```
@@ -0,0 +1,250 @@
+        # Clean up disconnected websockets
+        for ws in disconnected:
+            self.disconnect(ws)
+
+
+# Global connection manager instance
+manager = ConnectionManager()
+
+
+@router.websocket("/sentiment")
+async def websocket_sentiment_endpoint(
+    websocket: WebSocket, user: dict = Depends(get_current_user_websocket)
```

Ruff (0.13.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)